### ISSUE TITLE
Strange color scheme

### ISSUE THREAD

**Issue by dimitriirybakov** [Kronuz/Kronuz-Theme#5]
![image](https://user-images.githubusercontent.com/16699909/30162902-6ff739ee-93df-11e7-8bfd-5453bd41e37c.png)
![image](https://user-images.githubusercontent.com/16699909/30163338-44489980-93e1-11e7-9f09-37d8d8e7ca33.png)

config:
```
{
	"color_scheme": "Packages/User/SublimeLinter/Kronuz (SL).tmTheme",
	"theme": "Theme - Kronuz.sublime-theme"
}
```
What am i doing wrong?

**Comment by Kronuz** [comment]
What's strange about it?

**Comment by dimitriirybakov** [comment]
It's seriously differs from one on screenshot in README. Previously there was no that green colors. I reinstalled sublime and that happened

**Comment by Kronuz** [comment]
The theme was upgraded some time ago to better support more languages, most likely the changes added the green there for your language. I'm open about suggestions improving the theme colors. Do you have an screenshot of how it looked before or how you want it?

**Comment by Kronuz** [comment]
Is the README your reference? I'll try to get closer to it, but syntax changes in the languages could make it a bit difficult. I'm not all about that green either

**Comment by dimitriirybakov** [comment]
![image](https://user-images.githubusercontent.com/16699909/30164297-ab70299a-93e4-11e7-9dfd-116c5fb495c5.png)
That screenshot was made before update. It looked like screenshot in README

**Comment by dimitriirybakov** [comment]
![image](https://user-images.githubusercontent.com/16699909/30199650-50176340-947c-11e7-95be-f94f508a19f2.png)
Another example if it'll help you

**Comment by Kronuz** [comment]
Can you please check colors now? (I fixed quite a few in v1.0.8)

Also, if you're using JavaScript with modules, next SublimeCodeIntel (v3) will have support for ECMAScript with modules and other goodies :)

**Comment by dimitriirybakov** [comment]
Yeah, it's much better. Thanks!
